Whiskey/Functions/Publish-WhiskeyPSObject.ps1

```diff
@@ -0,0 +1,172 @@
+
+function Publish-WhiskeyPSObject
+{
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [Object] $Context,
+        
+        [Parameter(Mandatory, ParameterSetName='Module')]
+        [Management.Automation.PSModuleInfo] $ModuleInfo,
+        
+        [Parameter(Mandatory, ParameterSetName='Script')]
+        [PSCustomObject] $ScriptInfo,
+        
+        [String] $RepositoryName,
+        
+        [String] $RepositoryLocation,
+        
+        [String] $CredentialID,
+        
+        [String] $ApiKeyID
+    )
+
+    $commonParams = @{}
+    if( $VerbosePreference -in @('Continue','Inquire') )
+    {
+        $commonParams['Verbose'] = $true
+    }
+    if( $DebugPreference -in @('Continue','Inquire') )

```

Comment:
Remove these four lines. Setting `-Debug` switch will cause PowerShell to prompt if the user wants to stop or continue every time it encounters a `Write-Debug` statement.